agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -346,6 +373,24 @@ void monitor_thread::after_select(bool timeout)
             }
         }
 
+        // Process nl events
+        if (mon_hal_nl_events) {
+            if (read_ready(mon_hal_nl_events)) {
+                if (!mon_wlan_hal->process_nl_events()) {
+                    LOG(ERROR) << "process_nl_events() failed!";
+                    thread_last_error_code = MONITOR_THREAD_ERROR_NL_REPORT_PROCESS_FAIL;
+                    stop_monitor_thread();
+                    return;
+                }
+                clear_ready(mon_hal_nl_events);
+            }
+        } else {
```

Comment:
need to remove the else completely , we are entering here each after_select .
no need for this print at all.

Reply:
removed